desktop/cmp/dash/canvas/impl/DashCanvasView.ts

```diff
@@ -23,12 +23,13 @@ import {DashCanvasViewModel} from '../DashCanvasViewModel';
  *   - Track and trigger refreshes according to `DashCanvasViewSpec.refreshMode`.
  *   - Stretch its contents using a flex layout.
  *
- * @private
+ * @internal
```

Comment:
Sorry, I thought I had bombed all of these.  I'd suggest that we do so at the next opportunity (when we don't have any big TS PRs outstanding that could result in merge conflicts)